web/src/clipboard_handler.ts

```diff
@@ -0,0 +1,36 @@
+import * as blueslip from "./blueslip";
+import {decode_stream_topic_from_url} from "./hash_util";
+import {get_sub_by_id} from "./stream_data";
```

Comment:
Following the general pattern:
```suggestion
import * as stream_data from "./stream_data";
```